new-pipeline/src/dashboards.js

```diff
@@ -1077,7 +1077,13 @@ function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = buildDictionaryURL(metric, channel, description);
   var descr = metric;
   if (description && (description.length > 0)) {
-    descr = description;
+    if (description.length > 85) {
+      descr = description.substring(0, 85);
```

Comment:
Do we need this JavaScript change?
It would be great if we can do this just in CSS, so the dashboard can e.g. automatically show longer descriptions for larger screens.

Reply:
Actually, I did the same initially, but the "More details" option doesn't come without this JavaScript snippet. While,  CSS decreases the length of the long description.

Reply:
@georgf , Now, I got what you exactly want. I am working on it now. Hopefully, it will be ready in sometime. Actually, It requires very involved code change. I am fixing the bugs that I am getting.
Thanks!